examples/osgi-test-example-bnd/org.osgi.test.example.player.test/bnd.bnd

```diff
@@ -0,0 +1,17 @@
+# Build dependencies 
+-buildpath: \
+	${junit},\
+	${mockito},\
+	${osgitest},\
+	osgi.annotation;version='7.0',\
```

Comment:
Why specific versions here instead of latest? The buildpaths across the projects should have a consistent look.

Reply:
use lestest in all bnd.bnd files